client/my-sites/themes/themes-site-selector-modal.jsx

```diff
@@ -83,12 +83,12 @@ const ThemesSiteSelectorModal = React.createClass( {
 	 * but only if it also has a header, because the latter indicates it really needs
 	 * a site to be selected and doesn't work otherwise.
 	 */
-	wrapOption( option ) {
+	wrapOption( option, name ) {
 		return Object.assign(
 			{},
 			option,
 			option.header
-				? { action: themeId => this.showSiteSelectorModal( option, themeId ) }
+				? { action: themeId => this.showSiteSelectorModal( name, themeId ) }
```

Comment:
Arguably something to tackle separately, but, since `wrapOption` is called within the `render` stack, it would be cool to avoid instantiating functions here — preferably only passing data or, at the very least, preëxisting* bound methods.

*: can't resist them diæreses…